aedile-core/src/main/kotlin/com/sksamuel/aedile/core/LoadingCache.kt

```diff
@@ -175,6 +175,26 @@ class LoadingCache<K, V>(
       cache.synchronous().invalidateAll()
    }
 
+   /**
+    * Loads a new value for the key, asynchronously. While the new value is loading the
+    * previous value (if any) will continue to be returned by get(key) unless it is evicted.
+    *
+    * See full docs at [com.github.benmanes.caffeine.cache.LoadingCache.refresh].
+    */
+   fun refresh(key: K) {
+      cache.synchronous().refresh(key)
+   }
+
+   /**
+    * Loads a new value for each key, asynchronously. While the new value is loading the
+    * previous value (if any) will continue to be returned by get(key) unless it is evicted.
+    *
+    * See full docs at [com.github.benmanes.caffeine.cache.LoadingCache.refreshAll].
+    */
+   fun refreshAll(keys: Collection<K>) {
```

Comment:
Shouldn't the new methods be suspending, calling `await` and returning the value?

Reply:
Yes, I don't know what I was thinking?!
2.1.1 coming out

Reply:
... sorry to bug again, but what about the return value? :-)

Reply:
This is what happens when I do things after drinking :)
Try 2.1.2

Reply:
Thanks :-)